.github/workflows/ci.yaml

```diff
@@ -30,4 +31,13 @@ jobs:
         run: cargo clippy --all-targets
 
       - name: Fmt
-        run: cargo fmt --all -- --check
\ No newline at end of file
+        run: cargo fmt --all -- --check
+      - name: Fmt
+        run: cargo fmt --all -- --check
+
+      - name: Doc
+        run: |
+          cargo doc --workspace --all-features
+          cd macros
+          cargo doc --workspace --all-features
```

Comment:
I would argue this isn't strictly necessary as we re-export the public interface through the main crate, so it is being documented there.

Reply:
I don't think I fully understand this, you mean we run `cargo doc` somewhere else where this crate is used and we would catch in there the `rustdoc` errors?

Reply:
@cobward ?

Reply:
@cobward should I remove this?

Reply:
It's NABD, fine to leave this.